### PR TITLE
isPositiveZero, isNegativeZero - implementation + test skeleton method.

### PR DESCRIPTION
**Pull request by 5hanth**
- [x] Running `yarn lint` does not trigger any linter warnings
- [x] The test of the method you have fixed is passing
- [x] Add skeleton method and test cases for others to work on

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   floor: require("./floor"),
   increment: require("./increment"),
   indexOf: require("./indexOf"),
-  isPositiveZero : require("./isPositiveZero"),
+  iveZero : require("./isPositiveZero"),
```
Comment (andreasgruenh) [inline lib/index.js]: Is this name intentional? Why the change from `isPositiveZero` to `iveZero`?

**Comment by 5hanth** [comment]
@andreasgruenh : updated! pls review :)

**Comment by 5hanth** [comment]
@aaronjlech that was a typo. fixed!

File: test/take.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const { take } = require("../lib");
+
+test("Should return empty array if count is zero", () => {
+  expect(take([1,2,3],0)).toBe([]);
```
Comment (andreasgruenh) [inline test/take.test.js]: These functions should use `toEqual` and not `toBe`, as `toBe` checks for same identity.

Comment (5hanth) [reply test/take.test.js]: done !

**Comment by andreasgruenh** [comment]
@5hanth thanks for keeping up! Merged this now :)
